content/en/account_management/billing/_index.md

```diff
@@ -33,43 +33,53 @@ Datadog meters the count of IoT devices hourly. The billable count of IoT device
 
 For more information about IoT billing, see the [Datadog Pricing page][7].
 
-## Invoices
+## Plan details
 
-There are two choices for payment method:
-- Credit card
-- Invoicing (ACH, wire, or check)
+### Managing your payment method
 
-### Credit card
+The [**Payment Method**][8] section contains details on your payment options. 
 
-If you pay by credit card, receipts are available to [Administrators][8] for previous months under [Billing History][9]. For copies of your invoice, email [Datadog billing][10].
+{{< img src="account_management/billing/PaymentMethodOverview.png" alt="Payment method on the Plan page" style="width:90%;" >}}
 
-See [Credit Card Billing][11] for more details.
+**Edit Payment** provides options to manage payment methods. You can edit or remove cards, and request to change your payment method from card to invoice and vice versa. 
 
-### Invoicing
+{{< img src="account_management/billing/PaymentSettingsDetails.png" alt="Payment settings on the Plan page" style="width:90%;" >}}
 
-If you pay by check, ACH, or wire, invoices are emailed to the billing email addresses near the 10th business day of each month. If you need an additional copy, email [Datadog billing][10]. Details on where to remit payment can be found on the invoice.
+### Managing your billing contact details
 
-### Managing your payment method
+You can view your billing contact details on the [**Billing Contact Details**][8] section. 
 
-The [**Payment Method**][12] section contains details on your payment options. 
+{{< img src="account_management/billing/BillingContactDetailsOverview.png" alt="Billing contact details on the Plan page" style="width:90%;" >}}
 
-{{< img src="account_management/billing/PaymentMethodOverview.png" alt="Payment method on the Plan page" style="width:80%;" >}}
+Through **Edit Details**, you can add, edit or remove your billing address. You can also set specific email addresses to receive invoices.
 
-**Edit Payment** provides options to manage payment methods. You can edit or remove cards, and request to change your payment method from card to invoice and vice versa. 
+{{< img src="account_management/billing/BillingContactDetailsEdit.png" alt="Editing billing contact details on the Plan page" style="width:90%;" >}}
 
-{{< img src="account_management/billing/PaymentSettingsDetails.png" alt="Payment settings on the Plan page" style="width:80%;" >}}
+**Note**: The email address does not need to be a team member within Datadog. For example, you could use `invoices@yourcompany.com`.
```

Comment:
Just noting that as a general rule of thumb, we should be cautious when creating fictitious names (or in this case, an email address). For email addresses, the domain @example.com is specifically reserved for examples.